model/stacktrace_frame.go

```diff
@@ -24,23 +24,19 @@ type StacktraceFrame struct {
 	PreContext   []string `mapstructure:"pre_context"`
 	PostContext  []string `mapstructure:"post_context"`
 
-	SourcemapUpdated bool
-	SourcemapError   *string
-
 	ExcludeFromGrouping bool
+
+	Sourcemap Sourcemap
+}
+
+type Sourcemap struct {
+	Updated *bool
+	Error   *string
 }
 
-func (s *StacktraceFrame) Transform(config *pr.Config, service Service) common.MapStr {
+func (s *StacktraceFrame) Transform(config *pr.Config) common.MapStr {
```

Comment:
thinking of this as an api, what should happen if Transform is called more than once?

Reply:
Really good question. As the decision was made to overwrite the original values and not to create a new stacktrace frame out of it, running `Transform` multiple times would lead to a different output.